src/ukv_exceptions.py

```diff
@@ -0,0 +1,53 @@
+import json
+
+class UKVConfigurationException(Exception):
+    """Exception raised when problems loading the service configuration are encountered."""
+    def __init__(self, message='There we problems loading the configuration for the service. See logs.'):
+        self.message = message
+        super().__init__(self.message)
+
+class UKVKeyFormatException(Exception):
+    """Exception raised when a string presented as a Key is not correctly formatted."""
+    def __init__(self, message='The key is not properly formatted. See logs.'):
+        self.message = message
+        super().__init__(self.message)
+
+class UKVKeyNotFoundException(Exception):
+    """Exception raised when a valid Key is not found for the User."""
+    def __init__(self, message='Key not found for this user. See logs.'):
+        self.message = message
+        super().__init__(self.message)
+
+class UKVValueFormatException(Exception):
+    """Exception raised when a string presented as a Value is not correctly formatted as JSON."""
+    def __init__(self, message='The value is not properly formatted JSON. See logs.'):
```

Comment:
No need to say "See logs" for client errors since the users normally don't have access to the logs. Telling the users the proper format or why it's not correctly formatted can be more useful.